test/functional/fixtures/api/es-next/test-controller/test.js

```diff
@@ -35,6 +35,18 @@ describe('[API] TestController', () => {
         });
     });
 
+    describe('Assertions', () => {
+        it('Should track assertion without statements', () => {
```

Comment:
I guess the test should be moved to `test/functional/fixtures/api/es-next/assertions/test.js`
and renamed as `Should raise an error if assertion was called with method`.